### ISSUE TITLE
<img /> tags do not support using `width="100%"` or `width="50px"`.

### ISSUE THREAD

**Issue by mq2thez** [preactjs/legacy-compat#2]
Referencing an original issue from the Preact repo here:

https://github.com/preactjs/preact/issues/492

The core HTML spec doesn't allow for this stuff, and it appears that React v15 transforms them into style tags (100%) or removes the unit (50px).

**Comment by JoviDeCroock** [comment]
Hey,

I guess we could support that by adding something like the following code snippet [here](https://github.com/preactjs/legacy-compat/blob/dev/src/index.js#L94)

```js
if (vnode.type === 'img') {
  if (vnode.props.width) {
    if (vnode.props.width.endsWith('px')) {
      vnode.props.width = vnode.props.width.replace('px', '');
    } else if (vnode.props.width.endsWith('px')) {
      vnode.props.style = { ...vnode.props.style, width: vnode.props.width }
      delete vnode.props.width;
    }
  }

  // Similar for height
}
```

**Comment by JoviDeCroock** [comment]
Published as 0.1.6
